src/test/java/com/epam/tc/hw4/other/AttachmentUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.tc.hw4.other;
+
+import io.qameta.allure.Attachment;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class AttachmentUtil {
```

Comment:
`com.epam.tc.hw4.other.AttachmentUtil` is never used

Reply:
But it has use in Listener.class ._.

Reply:
No. Listener class use `com.epam.tc.hw4.utils.AttachmentUtil`

Reply:
You have 2 `AttachmentUtil` classes